src/randoop/DummyVisitor.java

```diff
@@ -9,13 +10,19 @@
   public void initialize(ExecutableSequence executableSequence) {
     // do nothing.
   }
-  
-  public void visitBefore(ExecutableSequence sequence, int i) {
-    // do nothing.
+
+  @Override
+  public void visitBeforeStatement(ExecutableSequence sequence, int i) {
+    // do nothing
+    
```

Comment:
This is from an Eclipse template that has a ${body-template} substitution that doesn't have a definition for a void method.